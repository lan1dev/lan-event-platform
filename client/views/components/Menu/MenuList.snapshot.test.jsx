import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';
import ManuList from './MenuList';

describe('ManuList', () => {
  let props;
  let wrapper;

  const Wrapper = () => {
    if (!wrapper) {
      wrapper = shallow(<ManuList {...props}>{props.children}</ManuList>);
    }
    return wrapper;
  };

  // Reset Component and Props before each test.
  beforeEach(() => {
    props = {};
    wrapper = undefined;
  });

  describe('without props', () => {
    it('Should match snapshot', () => {
      expect(toJson(Wrapper())).toMatchSnapshot();
    });
  });

  describe('with horizontal props', () => {
    it('Should match snapshot', () => {
      props = {
        horizontal: true
      };
      expect(toJson(Wrapper())).toMatchSnapshot();
    });
  });
});
