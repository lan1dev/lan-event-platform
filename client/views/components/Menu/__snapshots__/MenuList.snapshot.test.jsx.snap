// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ManuList with horizontal props Should match snapshot 1`] = `
.c0 {
  list-style-type: none;
  margin: 0 0 0 50px;
  padding: 0;
}

.c0 li {
  float: left;
  margin: 0 10px;
}

<ul
  className="c0"
/>
`;

exports[`ManuList without props Should match snapshot 1`] = `
.c0 {
  list-style-type: none;
  margin: 60px 0 0 50px;
  padding: 0;
}

<ul
  className="c0"
/>
`;
